### DISCUSSION TITLE
Silent Install

### DISCUSSION THREAD

**Discussion by lbd1277** [Open-Shell/Open-Shell-Menu#2060] [Q&A]
When doing a silent install is their a way to import current settings saved to a file ?

Currently using the below 
winget install -e --id Open-Shell.Open-Shell-Menu --accept-package-agreements --override "/norestart /quiet"

**Comment by ge0rdi** [comment]
Installer can't do that.
But after installation, you can run `"C:\Program Files\Open-Shell\StartMenu.exe" -xml <path-to-xml-with-settings>` to import your settings.
